api/build.sbt

```diff
@@ -13,11 +12,10 @@ version := "1.0"
 scalaVersion := "2.12.18"
 
 val ScalatraVersion = "3.1.1"
-val jettyVersion = "9.4.56.v20240826"
 val json4sVersion = "4.0.7"
-val logbackVersion = "1.2.13"
+val logbackVersion = "1.5.18"
```

Comment:
Older version of logback relied on SLF4j <1.7 which was incompatible with Jetty 12.